languages/elixir/exercises/concept/anonymous-functions/.docs/after.md

```diff
@@ -0,0 +1,42 @@
+Great work! [Anonymous functions][anon-fns] are commonly used in Elixir on their own, as return values, and as arguements in higher order functions such as `Enum.map/2`
+
+```elixir
+Enum.map([1, 2, 3], &(&1 + 1))
+# => [2, 3, 4]
+```
+
+### Key points
+
+- Functions are treated as first class citizens:
+  - Can be assigned to variables.
+  - Can be passed around like data as arguments and return values.
+  - Can be created dynamically.
+- [Anonymous functions][anon-fns] be created with the `fn` keyword:
+
+  ```elixir
+  varaible = fn param ->
+    param + 1
+  end
+  ```
+
+- [Anonymous functions][anon-fns] may be created with the [capture shorthand][capture]:
```

Comment:
I wondering if it would perhaps make sense to move this description after the "[Anonymous functions][anon-fns] be created with the `fn` keyword:" bit, which would then allow you to show both styles in the same code block:

```elixir
function_variable = fn param ->
  param + 1
end

anonymous_function_variable = &(&1 + 1)
```

I think the capture shorthand could use a little explanation on how it works (what is `&1` and such).

Reply:
I have added additional description